setup.sh

```diff
@@ -81,16 +85,6 @@ main() {
 	done
 	[ "${exit_now}" -ge 1 ] && exit "${exit_now}"
 
-	# Retrieve required packages if not present
```

Comment:
Keep opkg as it is provided by entware

Reply:
I think it is better to check for `fping` in the DEPENDENCIES then given that it's the default option and superior to iputils-ping. IMO iputils-ping should be dropped completely from cake-autorate

Reply:
I don't feel strongly either way about this

Reply:
> Not sure I agree. I don't think we need to / should install packages using opkg if they're missing. It's enough to report dependencies missing, no? Then user can decide to install. By the way, fping isn't strictly needed since user may instead use 'tsping' or 'iputils-ping'.

For Merlin users, you will (eventually) want to install all dependencies so there is a clean, automated installation through amtm one day. Otherwise, the friction to install will be too much for the less technical users of Merlin firmware.